pkg/integrations/v4/emitter/emitter.go

```diff
@@ -75,6 +75,23 @@ func (e *VersionAwareEmitter) Emit(definition integration.Definition, extraLabel
 		return err
 	}
 
+	// Agent creating the Host entity (and decorating it correctly in the backend) in secure forward with Custom Attributes: pkg/plugins/plugins_linux.go:46
+	// But in forward only there is no host entity and custom attributes are not being decorated.
+	// Here then we add CustomAttributes to extraLabels in case we are in that mode.
```

Comment:
nice example of good comment :)

Reply:
I know that everyone will forget why this hack is there :)